api/v1alpha3/virtualmachine_types.go

```diff
@@ -197,6 +197,19 @@ const (
 	// ManagerID on a VirtualMachine contains the UUID of the
 	// VMware vCenter (VC) that is managing this virtual machine.
 	ManagerID = GroupName + "/manager-id"
+
+	// +kubebuilder:validation:Enum=ImportVM;RegisterVM
+	//
```

Comment:
Remove the `//` between the kubebuilder annotation and the comment block. This prevents the kubebuilder annotation from becoming part of the GoDoc.